src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,16 +35,23 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService.getByUser(getUser(authentication));
         return orderMapper.mapToDto(orderService.completeOrder(cart));
     }
 
     @GetMapping
-    public List<OrderResponseDto> getOrderHistory(@RequestParam Long userId) {
-        return orderService.getOrdersHistory(userService.get(userId))
+    public List<OrderResponseDto> getOrderHistory(Authentication authentication) {
+        return orderService.getOrdersHistory(getUser(authentication))
                 .stream()
                 .map(orderMapper::mapToDto)
                 .collect(Collectors.toList());
     }
+
+    private User getUser(Authentication authentication) {
```

Comment:
no need for this private method. you can have 1 additional line of code by getting User with `userService.fingByEmail(authentication.getName()).orElseThrow( ... )` instead of 5 lines here

Reply:
okay :'(